scala3doc/src/dotty/dokka/site/common.scala

```diff
@@ -25,20 +24,19 @@ val apiPageDRI: DRI = mkDRI(packageName = "api", extra = "__api__")
 val defaultMarkdownOptions: DataHolder =
   new MutableDataSet()
     .setFrom(ParserEmulationProfile.KRAMDOWN.getOptions)
-    .set(
-      Parser.EXTENSIONS, List(
-        TablesExtension.create(),
-        TaskListExtension.create(),
-        AutolinkExtension.create(),
-        AnchorLinkExtension.create(),
-        EmojiExtension.create(),
-        YamlFrontMatterExtension.create(),
-        StrikethroughExtension.create()
-      ).asJava)
-    .set(
-      EmojiExtension.ROOT_IMAGE_PATH,
-      "https://github.global.ssl.fastly.net/images/icons/emoji/"
-    )
+    .set(Parser.INDENTED_CODE_BLOCK_PARSER, false)
```

Comment:
Do we intentionally disable indented code blocks?

Reply:
I copied the settings over from dotty-doc so I do not have a strong feeling aboout any of this options beside `ANCHORLINKS` ones.

Reply:
I think we should support them, unless we have a reason not to.

There's a separate topic here: linking to documentation from static site should be done with Wiki links, like in doc comments. This requires the WikiLink extension like [here](https://github.com/lampepfl/dotty/blob/master/scala3doc/src/dotty/dokka/tasty/comments/MarkdownParser.scala#L33) and some link-handling code that's already in MarkdownConverter [here](https://github.com/lampepfl/dotty/blob/master/scala3doc/src/dotty/dokka/tasty/comments/MarkdownConverter.scala#L87).